content/api-reference/queries/search-activities.mdx

```diff
@@ -70,3 +74,21 @@ When specifying multiple filters, the resulting criteria is the conjuction of al
   }
 }
 ```
+
+### Pagination and sorting
+
+Pagination and sorting are optional, **though we recommend not implementing this query without pagination**. Not using pagination means that the API will return all activities at once which might cause performance issues.
+
+```json
+{
+  "filters": {},
```

Comment:
```suggestion
```

The filters input itself is optional, it can be left out in this example since it's about pagination (if you like, depends what you are trying to show)